spark/src/main/scala/org/apache/mahout/sparkbindings/blas/ABt.scala

```diff
@@ -116,7 +118,12 @@ object ABt {
           // Empty combiner += value
           createCombiner = (t: (Array[K], Array[Int], Matrix)) =>  {
             val (rowKeys, colKeys, block) = t
-            val comb = new SparseMatrix(prodNCol, block.nrow).t
```

Comment:
I don't have any experimental evidence.. I just noticed it when using this as a reference for `FlinkOpABt`, And thought that maybe it was something that we should look at, since this has probably been in the codebase since before you added `MatrixFlavors`.

I would imagine that it couldn't hurt (and should do some good), in the case of two dense matrices (being combined and returned as sparse), but as you note its not going to catch the case of a sparse matrix turning dense because of a couple of dense rows/columns.  

I don't see this really adding much overhead to catch the `dense%*%dense` case. Should I commit it and then look into the densifying of sparse matrices in an other issue?